config/dev.php

```diff
@@ -17,4 +17,12 @@
         'region' => 'us-east-1',
         'endpoint' => 'http://localhost:4100',
     ],
+    'feature_rds' => false,
```

Comment:
This `config/*.php` files will really need to be moved into pillars for ease of editing by production.